src/scss/components/optional/_cuisines-widget.scss

```diff
@@ -23,113 +24,78 @@ $cuisinesWidget-gradient: linear-gradient(to top, rgba(0, 0, 0, 0.6), rgba(0, 0,
-
-           &:last-child {
-               margin-bottom: 0;
-           }
-
-           @include media('>=mid') {
-               min-height: 133px;
-               max-width: none;
-           }
-
-           @include media('>=narrow-mid') {
-               width: calc(50% - 8px);
-               margin-right: spacing(x2);
-               margin-left: 0;
-               margin-top: 0;
-
-               //for the grid of 2 elements for every second element not to have right margin
-               &:nth-child(even) {
-                   margin-right: 0;
-               }
-
-               //for the grid of 2 elements last row of elements not to have bottom margin
-               //(last child 0 margin is in initial styles above)
-               &:nth-last-child(2) {
-                   margin-bottom: 0;
-               }
-           }
-
-
-           @include media('>=mid-wide') {
-               min-height: 145px;
-               width: calc((100% - 32px) / 3);
-
-               &:nth-child(even) {
-                   margin-right: spacing(x2);
-               }
-
-               //Every third element will not have right margin + the last one
-               //(if the amount of items is not divisible by 3)
-               &:nth-child(3n+3),
-               &:last-child {
-                   margin-right: 0;
-               }
-
-               //for the grid of 3 elements last row of elements not to have bottom margin
-               //(last and second to last childs 0 margin are in styles above)
-               &:nth-last-child(3) {
-                   margin-bottom: 0;
-               }
-           }
-       }
-
-       .c-cuisinesWidget-grid {
-           display: flex;
-           flex-flow: row wrap;
-           justify-content: flex-start;
-           align-items: flex-start;
+                box-shadow: $cuisinesWidget-hoverBoxShadow;
+            }
 
            @include media('>=mid') {
-               max-width: 800px;
-               margin-left: auto;
-               margin-right: auto;
-           }
-
-           @include media('>=mid-wide') {
-               max-width: 1208px;
-           }
-       }
-   }
+                min-height: 133px;
+            }
+
+            @include media('>=mid-wide') {
+                min-height: 145px;
+            }
+        }
+
+        .c-cuisinesWidget-name {
+            position: absolute;
+            bottom: 0;
+            left: 0;
+            width: 100%;
+            padding: spacing(x2);
+            color: $cuisinesWidget-titleColour;
+            text-align: center;
+            background-image: $cuisinesWidget-gradient;
+
+            @include media('>=mid-wide') {
+                @include font-size(large, false);
+                padding: spacing(x3);
+            }
+        }
+
+        .c-cuisinesWidget-img {
+            vertical-align: bottom;
+            width: 100%;
+        }
+
+        /* autoprefixer grid: autoplace */
+        .c-cuisinesWidget-grid {
```

Comment:
Sorry I'm late to the game reviewing this one.  The naming looks a bit off here potentially looking at how the components used.

Dashes are for showing child elements, but it looks like this is the base component.  If so, it's probably better to name this `.c-cuisineGrid`, as `.c-cuisinesWidget-grid` is implying that it's a child of `.c-cuisinesWidget` (which I don't think it is looking at the docs site?).

Reply:
I've made a PR for this change: https://github.com/justeat/fozzie/pull/206